src/SFA.DAS.AANHub.Application/Models/MemberPreferenceModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.Models;
+
+public record MemberPreferenceModel(int PreferenceId, string Value)
```

Comment:
why is value a string here? it should be boolean

Reply:
Changed